lib/include.js

```diff
@@ -877,9 +877,15 @@ Inclusion.include = function(objects, include, options, cb) {
         defineCachedRelations(obj);
         obj.__cachedRelations[relationName] = null;
       }
-      filter.where[relation.keyTo] = {
-        inq: uniq(targetIds),
-      };
+      if (!filter.where[relation.keyTo]) {
+        filter.where[relation.keyTo] = {
+          inq: uniq(targetIds),
+        };
+      } else {
```

Comment:
the test should be:
`} else if (filter.where[relation.keyTo] && typeof filter.where[relation.keyTo] !== 'object') {`

since where filter on id could be of type {inq: [ids..]}, or any type among nin, neq, regex, etc.
in that case you do not want to use inq as in line 886 (`inq: [filter.where[relation.keyTo]]`).